CodingChallenges/CC_116_Lissajous/P5/sketch.js

```diff
@@ -0,0 +1,99 @@
+// Daniel Shiffman
+// http://youtube.com/thecodingtrain
+// http://codingtra.in
+// Javascript transcription: Chuck England
+
+// Coding Challenge #116: Lissajous Curve Table
+// https://youtu.be/--6eyLO78CY
+
+// p5.js version:
+// https://editor.p5js.org/codingtrain/sketches/BJbj5l3Y7
+
+let angle = 0
```

Comment:
Would you mind adding semi-colons? (Don't tell MPJ)

Reply:
Done. (my eyes, they burn, lol)